core/src/main/java/org/elasticsearch/cluster/routing/ShardRouting.java

```diff
@@ -420,7 +420,7 @@ void initialize(String nodeId) {
     void relocate(String relocatingNodeId) {
         ensureNotFrozen();
         version++;
-        assert state == ShardRoutingState.STARTED : this;
+        assert state == ShardRoutingState.STARTED : "current shard has to be started inorder to relocated " + this;
```

Comment:
"current shard has to be started inorder to relocated " -> "current shard has to be started in order to be relocated "